create_infotexts.py

```diff
@@ -132,8 +136,8 @@ def load_json_metadata(infile):
 
 
 def create_people_mapping_wikitable(people_mapping):
-    """
-    Transform json style people mapping to wikitable
+    """Transform json style people mapping to wikitable.
+
     :param people_mapping: json dictionary
```

Comment:
I guess that technically this is just a `dict` (you don't actually load the json file here)